scripts/notifications.coffee

```diff
@@ -213,4 +252,30 @@ e.g. --message Hello World --channel general tech --> "Hello World" used as --me
       # Add necessary configurations to vars
       vars.plugins[medium] = varsTemp[medium]
 
-    doRequest(vars, res)
+    # Verify message:
+    robot.messageRoom CONTROL_CHANNEL_NAME, "New notification pending. How's it look? (type `botgt notify approve` or `botgt notify abort`)"
+    robot.messageRoom CONTROL_CHANNEL_NAME, configToLog(logDict)
+    robot.brain.set('pending', JSON.stringify(vars))
+    robot.brain.set('sender', res.envelope.user.name)
+    # robot.messageRoom VERIFY_CHANNEL_NAME, {
+    #   text: configToLog(logDict),
+    #   attachments: [{
+    #     text: "Send message?", 
+    #     actions: [
+    #       {
+    #           "name": "choice",
+    #           "text": "Approve",
+    #           "type": "button",
+    #           "value": "approve"
+    #       },
+    #       {
+    #           "name": "choice",
+    #           "text": "Abort",
+    #           "type": "button",
+    #           "value": "cancel"
+    #       }
+    #     ]
+    #   }]
+    # }
+    res.send ''
```

Comment:
Why res.send nothing here?

Reply:
ah can I just not include anything, that's cool too, I'll do that